parsl/addresses.py

```diff
@@ -9,36 +9,52 @@
 logger = logging.getLogger(__name__)
 
 
-def address_by_route():
+def address_by_route() -> str:
+    """Finds an address for the local host by querying the local routing table
+       for the route to Google DNS.
+
+       This will return an incorrect value when the internet-facing address is
```

Comment:
This may not return a correct value when the network doesn't give an externally accessible address ?